Asterix/WSC_functions.py

```diff
@@ -539,53 +517,44 @@ def createdifference(input_wavefront,
     Difference : 3D array
         Cube with image difference for each probes. Use for pair-wise probing
     -------------------------------------------------- """
-    Ikmoins = np.zeros((testbed.dim_im, testbed.dim_im))
-    Ikplus = np.zeros((testbed.dim_im, testbed.dim_im))
+    if wavelength == None:
+        wavelength = testbed.wavelength_0
+
     Difference = np.zeros((len(posprobes), dimimages, dimimages))
```

Comment:
No. In this case dimimages is the size after resmapling so it's not dim_in. I decided on purpose not to put any resampling information in the testbed class, because these are correction / estimation information, not testbed direclty. However, I agree with you that some of these param will disapepar when this function will be included in an "estimation" class